data/hai missions.txt

```diff
@@ -915,7 +915,7 @@ mission "Expanding Business [7]"
 	on complete
 		event "shipyard on greenwater" 180
 		conversation
-			`	Construction on the shipyard begins almost immediately after the supplies are unloaded from the freighters, starting with the demolition of the rest of the park. A downtrodden looking crowd gathers to watch the demolition, but Turner ensures to you that there are plenty of other parks in the city for them to go to.`
+			`	Construction on the shipyard begins almost immediately after the supplies are unloaded from the freighters, starting with the demolition of the rest of the park. A downtrodden looking crowd gathers to watch the demolition, but Turner assures you that there are plenty of other parks in the city for them to go to.`
```

Comment:
While you're here: `downtrodden-looking`

Reply:
Thanks for contributing! If you want to talk more about the game and how to make it better, please join us on discord at https://discord.gg/ZeuASSx.